src/ophys_etl/transforms/mesoscope_2p/conversion_utils.py

```diff
@@ -34,6 +40,30 @@ def volume_to_h5(h5fp, volume, dset_name="data", page_block_size=None,
             i += page_block_size
 
 
+def volume_to_h5(h5fp, volume, dset_name="data", page_block_size=None,
+                 **h5_opts):
+    """
+    Write a tiff volume to an HDF5 file.
+
+    h5fp can either be a string indicating the path to the HDF5 file
```

Comment:
do we need/use both of these options?

Reply:
It's unclear. I needed this to accept a string for the testing, but didn't want to break any modules that might use this with an h5 file. Is it worth trying to refactor my test to not need it to accept a string?

Reply:
you should not add functionality to make your tests work. you need to make your tests test the functions as they are used.

Reply:
Fixed. I reverted this function to its previous version.